MAUI/DataForm/right-to-left.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Right To Left FlowDirection in .NET MAUI DataForm control | Syncfusion
+description: Learn about Right To Left support in Syncfusion .NET MAUI DataForm (SfDataForm) control in mobile and desktop applications from a single shared codebase
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Right To Left Flow Direction in .NET MAUI DataForm (SfDataForm)
+
+`SfDataForm` supports changing the flow direction of text to the right-to-left direction by setting the `FlowDirection` to `RightToLeft`
```

Comment:
`SfDataForm` supports changing the flow direction of the text to the right-to-left direction by setting the `FlowDirection` to `RightToLeft.`

Reply:
content changed